core/platform/auth/firebase_auth_services.py

```diff
@@ -462,10 +472,11 @@ def _get_auth_claims_from_session_cookie(cookie):
             with _firebase_admin_context():
                 return _create_auth_claims(
                     firebase_auth.verify_session_cookie(cookie))
+        # NOTE: We're OK with cookies becoming obsolete with time, and will
+        # suppress the errors related to this. See the constant for details.
         except EXPECTED_SESSION_COOKIE_EXCEPTIONS:
             # NOTE: logging.exception appends the stack trace automatically.
             logging.exception('User session has ended and must be renewed')
```

Comment:
Maybe explain, in a code comment: it isn't an error for a session cookie to be missing, it just means the request isn't authenticated.